src/SprykerEco/Zed/PunchoutCatalogs/Communication/Table/PunchoutCatalogsConnectionsTable.php

```diff
@@ -196,6 +197,10 @@ protected function mapPunchoutCatalogConnection(PgwPunchoutCatalogConnection $pu
             $punchoutCatalogConnection->getFormat(),
             'label-secondary'
         );
+        $punchoutCatalogConnectionRow[static::COL_TYPE] = $this->generateLabel(
+            $punchoutCatalogConnection->getType(),
+            'label-secondary'
```

Comment:
Why it's not a constant?

Reply:
Just checked all existing `Table.php`'s, seams everywhere it's done without constant, so I would keep it as is